### PR TITLE
ZCS-633: Universal UI- Move the refresh button to the top header

### PR DESCRIPTION
**Pull request by kalpeshhpatel**
* Animated refresh button for some predefined time `1500ms` when refresh button is clicked.

https://jira.corp.synacor.com/browse/ZCS-633

### CODE REVIEW

File: WebRoot/skins/_base/base4/skin.css [hunk 1]
```diff
@@ -1356,3 +1356,17 @@ div[name='_mergeBtnContainerId'] .ZCalToggleBtnSelected .ZWidgetTitle {
...
+@keyframes spin {
+    0% {
+      @transform@: rotate(0deg);
+    }
+    100% {
+      @transform@: rotate(360deg);
+    }
+}
+
+.ZmSpinElement {
```
Comment (silentsakky) [inline WebRoot/skins/_base/base4/skin.css]: can we move this to zm.css?

Comment (kalpeshhpatel) [reply WebRoot/skins/_base/base4/skin.css]: Sure, we can. Let me move it.

Comment (kalpeshhpatel) [reply WebRoot/skins/_base/base4/skin.css]: Moved: https://github.com/Zimbra/zm-web-client/pull/210/commits/78c4af87182fc5668dc6ffd0d206a382e8044f4b

Thanks

Comment (silentsakky) [reply WebRoot/skins/_base/base4/skin.css]: Thanks

**Pull request merged**